MAUI/NavigationDrawer/Toggle-Animations.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: Setting Toggle Animations in .NET MAUI Navigation Drawer | Syncfusion
+description: Learn here all about Setting Toggle Animations support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+# Setting Toggle Animations in .NET MAUI Navigation Drawer
+
+The drawer toggling animation can be changed using `Transition` property and it can be set to three different values. They are
```

Comment:
The drawer toggling animation can be changed using the `Transition` property, and it can be set to three different values. Which are:

Reply:
Changed